src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesSlave.java

```diff
@@ -216,6 +219,7 @@ public String getPodName() {
 
     private String remoteFS;
 
+    @SuppressFBWarnings(value = "NM_CONFUSING", justification = "Naming confusion with a getRemoteFs method, but the latter is deprecated.")
```

Comment:
A bit surprised that this is raised on a method that gets overridden, but well...

Reply:
:shrug: